routingpy/routers/__init__.py

```diff
@@ -33,15 +32,15 @@
     "here": HereMaps,
     "heremaps": HereMaps,
     "mapbox_osrm": MapboxOSRM,
-    "mapbox_valhalla": MapboxValhalla,
     "mapbox-osrm": MapboxOSRM,
-    "mapbox-valhalla": MapboxValhalla,
     "mapbox": MapboxOSRM,
     "mapboxosrm": MapboxOSRM,
-    "mapboxvalhalla": MapboxValhalla,
     "openrouteservice": ORS,
+    "opentripplanner": OpenTripPlannerV2,
```

Comment:
Do we really need several aliases? I find it more bearable over time to provide the right key from the start.

Reply:
We don't _need_ it, but typing "otp" instead of "opentripplanner_v2" is nicer IMO. The particular line you're indicating is indeed a bit more useless

Reply:
Yes, I agree with you about the abbreviations. I was thinking more of the version with underscore or hyphen. But no biggie, it was just a detail.